sno/gpkg.py

```diff
@@ -207,13 +176,13 @@ def pk(db, table):
 
 
 def geom_cols(db, table):
-    q = db.cursor().execute(
+    q = db.execute(
```

Comment:
Yup - happy to switch everything to use sqlalchemy functions. But, it's not done in this PR. I might do it after porting PostGIS to sqlalchemy, since then I can port common WC code to use sqlalchemy properly.